lametro/templates/committee.html

```diff
@@ -132,7 +135,8 @@ <h4><i class="fa fa-user"></i> Chief Executive Officer</h4>
                 null,
                 { "sType": "num-html" },
                 null
-            ]
+            ],
+            "order": []
```

Comment:
This attribute prevents datatables from adding a sort icon to the first(empty) `th` used for the images

#### Before
<img width="482" alt="image" src="https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/d76329eb-a325-4409-b36c-027a11545320">

---

#### After 
<img width="487" alt="image" src="https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/561eb9a7-2093-4b75-8a63-5417d2cc9a28">